ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -271,6 +272,9 @@ public void processPriorCommands(final PersistentQueryCleanupImpl queryCleanup)
       final List<QueuedCommand> compatibleCommands = checkForIncompatibleCommands(restoreCommands);
 
       LOG.info("Restoring previous state from {} commands.", compatibleCommands.size());
+      if (compatibleCommands.size() > COMMAND_TOPIC_THRESHOLD_LIMIT) {
+        LOG.warn("Command topic size exceeded {} commands.", COMMAND_TOPIC_THRESHOLD_LIMIT);
```

Comment:
log both for better debugging experience. Finding nearby docs is not so easy experience.
```suggestion
        LOG.warn("Command topic size exceeded. [commans={}, threshold={}"],  compatibleCommands.size(), COMMAND_TOPIC_THRESHOLD_LIMIT);
```